src/Integration/Binding/BindingProcessImpl.cs

```diff
@@ -241,64 +219,8 @@ public void SilentSaveSolutionIfDirty()
 
         #region Private methods
 
-        private void InformAboutFilteredOutProjects()
```

Comment:
We used to log a list of the "unsupported" projects i.e. those that use languages we don't support.
There's no point now we fetch settings for all languages.

Also, `Secrets` analysis is run against every text file, so there probably isn't such a thing as a project with an unsupported language type any more.